### DISCUSSION TITLE
Integrate Nuclei to scanner

### DISCUSSION THREAD

**Discussion by Caratpine** [projectdiscovery/nuclei#3302] [Q&A] [answered]
Hi, I have a scanner that feeds a large number of assets into Nuclei for scanning.
Currently, I use Go `exec.Command` to implement this way, but this will spawn many sub processes.
I want to integrate Nuclei into the scanner and run it as a goroutine. Is there any way? thanks.

**Comment by tarunKoyalwar** [comment, marked as answer]
@Caratpine , it is possible and there is also a getting started example at https://github.com/projectdiscovery/nuclei#using-nuclei-from-go-code
